backend/src/api/cycle.js

```diff
@@ -4,6 +4,8 @@ const { directorsOnly, errorWrap } = require('../middleware')
 const router = express.Router()
 
 // Get all cycles (previous and current)
+// TODO: when authentication server is integrated, only show the cycles that
```

Comment:
let's link ticket numbers to these TODO items!

Reply:
ya np

Reply:
https://github.com/hack4impact-uiuc/h4i-recruitment/issues/305